app-emulation/fuse/fuse-1.5.1.ebuild

```diff
@@ -50,15 +50,18 @@ src_configure() {
 		guiflag="--with-fb"
 	fi
 
-	econf \
-		--without-win32 \
-		${guiflag} \
-		$(use_with alsa) \
-		$(use_with ao libao) \
-		$(use_with gpm) \
-		$(use_with joystick) \
-		$(use_enable joystick ui-joystick) \
-		$(use_enable memlimit smallmem) \
-		$(use_with png) \
+	local myconf=(
+		--without-win32
+		${guiflag}
```

Comment:
Now you can kill the extra variable, and just append to `myconf` inside the conditionals.